pkgs/development/python-modules/seabreeze/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv
+, fetchFromGitHub
+, python3Packages
+}:
+
+## Usage
+# In NixOS, simply add the `udev` multiple output to services.udev.packages:
+#   services.udev.packages = [ pkgs.python3Packages.seabreeze.udev ];
+
+python3Packages.buildPythonPackage rec {
+  pname = "seabreeze";
+  version = "v60569e";
+
+  src = fetchFromGitHub {
+    owner = "ap--";
+    repo = "python-seabreeze";
+    rev = "60569e7b2c0e766d1f00267bb4a752333df4eb77";
+    sha256 = "00jf7ycy34wsgv2sv3jwqg822s83nc1psvakn3ql1dk0w14rvz23";
+  };
+
+  outputs = [ "out" "udev" ];
+
+  postInstall = ''
+    mkdir -p $udev/lib/udev/rules.d
+    cp misc/10-oceanoptics.rules $udev/lib/udev/rules.d/10-oceanoptics.rules
+  '';
+
+  doCheck = false;
```

Comment:
Why are these disabled? Are there none?

Reply:
underlying c libraries are tested and fail, (c libs are used with anaconda, which we don't care about).

Reply:
Please add a comment explaining that.

Often maintainers may try to enable tests and having any context as to why they were disabled is helpful.

Reply:
Sorry buddy, no I won't, anyone who wants to enable this may make a change and test it themselves.